src/components/v5/shared/StatusText/types.ts

```diff
@@ -0,0 +1,15 @@
+import { MessageDescriptor } from 'react-intl';
+
+export interface StatusTextProps {
+  status: (typeof STATUS_TYPES)[keyof typeof STATUS_TYPES];
+  title: string | MessageDescriptor;
+  withIcon?: boolean;
+  fontSizeClassName?: string;
```

Comment:
You can pass any class here, so maybe it would be better to rename it to textClassName or something like this.

Reply:
Renamed